src/services/textChanges.ts

```diff
@@ -682,7 +682,7 @@ namespace ts.textChanges {
             return {
                 indentation,
                 prefix: (insertLeadingComma ? "," : "") + this.newLineCharacter,
-                suffix: insertTrailingComma ? "," : ""
+                suffix: insertTrailingComma ? "," : isInterfaceDeclaration(node) && isEmpty ? ";" : ""

```

Comment:
The `;` depends on the preferences, right?

Reply:
I suppose that nowhere else in TypeScript do we generate new code for a semicolonless style.